nwbinspector/checks/behavior.py

```diff
@@ -0,0 +1,9 @@
+from pynwb.behavior import SpatialSeries
+
+from ..register_checks import register_check, Importance, InspectorMessage
+
+
+@register_check(importance=Importance.CRITICAL, neurodata_type=SpatialSeries)
+def check_spatial_series_dims(spatial_series: SpatialSeries):
+    if len(spatial_series.data.shape) > 1 and spatial_series.data.shape[1] > 3:
+        return InspectorMessage("SpatialSeries should have either 2 columns (x,y) or 3 columns (x,y,z).")
```

Comment:
What about 1 column (x) for linear tracks?

Reply:
yes, those will be false with `if len(spatial_series.data.shape) > 1`, so they will pass the check

Reply:
I added a test for this

Reply:
Right, I just mean in the message string itself (sorry, I selected only that one line when I made the comment so I thought it would only highlight that line only)

Reply:
I.e., "SpatialSeries should have either 1 column (x), 2 columns (x,y)..."

Reply:
ah ok you're right